projects/i18n/interfaces/language.ts

```diff
@@ -42,6 +42,9 @@ export interface LanguageKit {
         'HH:MM:SS': string;
         'HH:MM:SS.MSS': string;
     };
+    dateTexts: {
+        'dd.mm.yyyy': string;
```

Comment:
Please also add it to the newly contributed Vietnamese translation

Reply:
done